cpp/data/installer/apparmor/apache2

```diff
@@ -0,0 +1,30 @@
+# vim:syntax=apparmor
+#include <tunables/global>
+
+/usr/sbin/apache2 flags=(attach_disconnected) {
+  #include <abstractions/base>
+
+  #include <abstractions/apache2-common>
+  #include <abstractions/mysql>
+  #include <abstractions/php>
+  #include <abstractions/postfix-common>
+
+  # apache2 basics
+  /etc/** r,                    # apache.conf, also /etc/ssl and others
```

Comment:
Maybe we should be more specific here.  For example it can be dangerous to be able to read the contents of /etc/passwd

Reply:
done